core/src/main/scala/com/salesforce/op/stages/impl/feature/TextTokenizer.scala

```diff
@@ -162,30 +162,84 @@ object TextTokenizer {
     toLowercase: Boolean = ToLowercase,
     minTokenLength: Int = MinTokenLength
   ): TextTokenizerResult = {
-    text match {
-      case SomeValue(Some(txt)) =>
-        val language =
-          if (!autoDetectLanguage) defaultLanguage
-          else {
-            languageDetector
-              .detectLanguages(txt)
-              .collectFirst { case (lang, confidence) if confidence > autoDetectThreshold => lang }
-              .getOrElse(defaultLanguage)
-          }
-        val lowerTxt = if (toLowercase) txt.toLowerCase else txt
+    val language =
+      if (!autoDetectLanguage) defaultLanguage
+      else {
+        languageDetector
+          .detectLanguages(textString)
+          .collectFirst { case (lang, confidence) if confidence > autoDetectThreshold => lang }
+          .getOrElse(defaultLanguage)
+      }
+    val lowerTxt = if (toLowercase) textString.toLowerCase else textString
 
-        val sentences = sentenceSplitter.map(_.getSentences(lowerTxt, language))
-          .getOrElse(Seq(lowerTxt))
-          .map { sentence =>
-            val tokens = analyzer.analyze(sentence, language)
-            tokens.filter(_.length >= minTokenLength).toTextList
-          }
-        TextTokenizerResult(language, sentences)
-      case _ =>
-        TextTokenizerResult(defaultLanguage, Seq(TextList.empty))
+    val sentences = sentenceSplitter.map(_.getSentences(lowerTxt, language))
+      .getOrElse(Seq(lowerTxt))
+      .map { sentence =>
+        val tokens = analyzer.analyze(sentence, language)
+        tokens.filter(_.length >= minTokenLength).toTextList
```

Comment:
Why are we only keeping tokens with length > `minTokenLength ` ?

Reply:
This was existing behavior. It's a configurable parameter (defaulting to 1), so is not required.